packages/vue/src/index.ts

```diff
@@ -177,10 +177,11 @@ const Frontegg: PluginObject<PluginOptions> | any = {
     }, 10);
 
     const isAuthRoutes = (path: string) => {
+      const pathname = new URL(path, window.location.origin).pathname
```

Comment:
Previously is was extracted from the currentRoute - how can we make sure it's not a breaking change for other customers? @frontegg-david

Reply:
This is still extracted from `currentRoute` just for vue router 4 it is taken from `value. fullPath`, why do we not just take the `path` and not `fullpath `and avoid this line? @frontegg-david

Reply:
this fixes the isAuthRoute bugs.